aiohttp_devtools/runserver/main.py

```diff
@@ -2,44 +2,48 @@
 import os
 from multiprocessing import set_start_method
 
-from watchdog.observers import Observer
-
 from ..logs import rs_dft_logger as logger
 from .config import Config
 from .serve import HOST, check_port_open, create_auxiliary_app
-from .watch import AllCodeEventHandler, LiveReloadEventHandler, PyCodeEventHandler
+from .watch import AppTask, LiveReloadTask
 
 
-def run_app(app, observer, port, loop):
+def run_app(app, port, loop):
     handler = app.make_handler(access_log=None, loop=loop)
-    server = loop.run_until_complete(loop.create_server(handler, HOST, port))
+
+    co = asyncio.gather(
+        loop.create_server(handler, HOST, port),
```

Comment:
One note: better to start serving *after* processing `app.startup` signal.
Otherwise there is a small chance for handling first incoming request on not initialized application.

Reply:
can we switch to use aiohttp's own `run_app` method?

Reply:
Think I'll work on this on anther PR.

Reply:
Not sure. Maybe. Need to dig in code.
BTW for next release I want to split `run_app` functionality into a new class and it's methods.
The idea is providing a tool for usages like this.

Reply:
ok, I'll keep like this for now.